debug/TorrentClient/test_get_message.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+from message import Message, \
+                    KeepAlive, \
+                    Choke, \
+                    UnChoke, \
+                    Interested, \
+                    NotInterested, \
+                    Have, \
+                    BitField, \
+                    Request, \
+                    Piece, \
+                    Cancel, \
+                    Port
+
+class TestGetMessage(unittest.TestCase):
+
+    def test_keep_alive(self):
+    	message = Message.get_message('keep-alive')
+    	self.assertEqual(message, KeepAlive())
```

Comment:
The way Adam showed me was like this:
def test_get_message_keepalive(self):

       """ Test keepalive messages are returned correctly. """



       msg = Message.get_message('keep-alive')

       self.assertEqual(isinstance(msg, KeepAlive), True)



   def test_get_message_keepalive_length(self):

       """ Test keepalive messages are  the correct length """



       msg = Message.get_message('keep-alive')

       self.assertEqual(msg.length, 0)



   def test_get_message_keepalive_to_bytes(self):

       msg_b = Message.get_message('keep-alive').to_bytes()

       self.assertEqual(msg_b, b'\x00\x00\x00\x00')

This would be everything we would need for the testcase keep-alive instance